### ISSUE TITLE
[Suggestion] Randomise the time Monika sleeps/stays awake

### ISSUE THREAD

**Issue by IsItBadToMakeAGitHubAccountJustForThis** [Monika-After-Story/MonikaModDev#1166]
Self descriptive.
I haven't seen the code, responsible for Monika getting tired, but I am guessing that both the amount of time she can stay awake and the amount of time she needs to rest are constant.
I suggest we allow them fluctuate within certain margins to make it more realistic.
I know that this will probably go unnoticed by most of the players and is probably too much effort for such a minor feature, but I believe it wouldn't be too hard to implement.

**Comment by ThePotatoGuy** [comment]
Yeah makes sense, probably some distribution between 5-9/10 hours

**Comment by AnIdiotsavant** [comment]
how bout her lying the desk with her arms around her face. You know.. what a student usually looks like in calculus. Asleep on their desk

**Comment by ThePotatoGuy** [comment]
I'll have you know that I was _always_ awake in calc.

**Comment by AnIdiotsavant** [comment]
I'll have *you* know I always fall asleep in calc
